services/management/commands/services_import/units.py

```diff
@@ -364,6 +364,20 @@ def _import_unit(syncher, keyword_handler, info, dept_syncher,
         obj_changed = True
         obj.geometry = obj.location
 
+    if obj.municipality_id is None and obj.location is not None:
+        try:
+            div = AdministrativeDivisionGeometry.objects.select_related('division')\
+                .filter(boundary__contains=obj.location, division__type_id=1)
```

Comment:
Better to check division__type__type = 'muni'
Also add division__type to select_related